events/git-and-github.md

```diff
@@ -0,0 +1,126 @@
+# Git & Github Workshop
+
+## Aim of the Workshop
+
+At the end of the workshop, make sure that audience are comfortable with the following
+
+- Why Version Control Systems are need?
+- Usage of basic Git commands to track their projects - Adding and Committing
+- Be able to push and pull code from Github
+- Be able to send a PR to an Open Source Repository
+
+## Before the workshop
+
+Share an instructions file like [this](https://github.com/kossiitkgp/git-and-github-workshop-2020/blob/main/installation.md), along with the poster of the workshop. The instructions should contain
+
+- How to install Git for Windows, Linux, Mac seperately
+- For Windows users, make that they install `git bash`
+- How to verify that git is installed
+- Instructions on how to create a Github Account
+
+In the post mention the fact that we will be available for installation issues before 15 minutes of the workshop. Help with the installation issues before the workshop begins.
+
+## Flow of the workshop
+
+-1. **How will the workshop proceed**
+
+   Explain that Git and Github are important parts of Software Development. Tell them how the workshop will proceed like when should audience ask queries. Tell the audience that doing hands-on by themselves along with the workshop will help them learn better. Encourage them to ask questions, as some of the Git concepts are non-intuitive.
+
+0. **Terminal Commands**
```

Comment:
Could you fix the indentation of the points? Each paragraph looks indented

Reply:
Done
Thanks for pointing it out!

The very first paragraph was not intended because it was of serial number `-1`. Negative numbers are not properly intended. I tried 15 mins to intend, but failed. Changed to order starting from `0`

Reply:
*indented